frontend/ui/widget/bookstatuswidget.lua

```diff
@@ -27,6 +27,8 @@ local util = require("util")
 local _ = require("gettext")
 local Screen = require("device").screen
 local template = require("ffi/util").template
+local Device = require("device")
```

Comment:
Adding Device here requires putting it alphbetically in order (on line 5).

That in turn means ` local Screen = require("device").screen` should be changed to ` local Screen = Device.screen`